packages/shared/styles/components/molecules/SfNotification.scss

```diff
@@ -60,6 +60,8 @@
     top: var(--notification-close-top, var(--spacer-xs));
     right: var(--notification-close-right, var(--spacer-xs));
     cursor: pointer;
+    background: transparent;
```

Comment:
maybe we should add some modifier to button ? like `--icon` - now we have many repeated properties to remove properties, wdyt @justyna-13 @pspaczek  ?

Reply:
Sounds good for me :) we have some places where we change the same properties for button with icons

Reply:
button--unstyled? Then you can use it for all case, not only for icon. @martaradziszewska, @justyna-13?

Reply:
button--unstyled === reset button styles

Reply:
https://github.com/DivanteLtd/storefront-ui/issues/1116 issues